### PR TITLE
[Feature] create account controller

### PR DESCRIPTION
**Pull request by chandraarigunawan**
## Summary
Create account controller

## Test Plan
test manual in local

### CODE REVIEW

**Review by feechanz — APPROVED** [review]

File: spring-command-pattern-web/src/main/java/com/idspring/commandpattern/controller/AccountController.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import reactor.core.publisher.Mono;
+import reactor.core.scheduler.Schedulers;
+
+@RestController
+@RequestMapping("/accounts")
+public class AccountController {
+
```
Comment (nellacristiani) [inline spring-command-pattern-web/src/main/java/com/idspring/commandpattern/controller/AccountController.java:16]: test aja

File: spring-command-pattern-web/src/main/java/com/idspring/commandpattern/controller/AccountController.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+@RestController
+@RequestMapping("/accounts")
+public class AccountController {
+    
+    @RequestMapping(value = "/", method = RequestMethod.GET,
+    produces = MediaType.APPLICATION_JSON_VALUE)
+    public Mono<Response<String>> getaccount() {
+        return Mono.just(Response.ok("account"))
+                .subscribeOn(Schedulers.elastic());
+    }
```
Comment (nellacristiani) [inline spring-command-pattern-web/src/main/java/com/idspring/commandpattern/controller/AccountController.java:22]: test lagi

**Review by nellacristiani — CHANGES_REQUESTED** [review]

File: spring-command-pattern-web/src/main/java/com/idspring/commandpattern/controller/AccountController.java (hunk 1 above)
Comment (chandraarigunawan) [reply spring-command-pattern-web/src/main/java/com/idspring/commandpattern/controller/AccountController.java:16]: its already done

File: spring-command-pattern-web/src/main/java/com/idspring/commandpattern/controller/AccountController.java (hunk 2 above)
Comment (chandraarigunawan) [reply spring-command-pattern-web/src/main/java/com/idspring/commandpattern/controller/AccountController.java:22]: fixed

**Review by nellacristiani — APPROVED** [review]

**Pull request merged**